src/pages/inventory/[[...section]].tsx

```diff
@@ -586,8 +586,8 @@ const Inventory = () => {
     );
 
     switch (section) {
-      case "hidden":
-        return [hidden, totals, updates, "No hidden listings 🙂"] as const;
+      case "staked":
+        return [staked, totals, updates, "No staked listings 🙂"] as const;
```

Comment:
Let's change this back so we don't have to change the GraphQL query.

```suggestion
        return [hidden, totals, updates, "No staked listings 🙂"] as const;
```